packages/history-utility/src/index.ts

```diff
@@ -23,11 +23,14 @@ export type HistoryNode<T> = {
   updatedAt?: Date;
 };
 
+type EmptyWIP = Record<string, never>;
+const EmptyWIP: EmptyWIP = {};
+
 export type History<T> = {
   /**
    * field for holding sandbox changes; used to avoid infinite loops
    */
-  wip?: Snapshot<T>;
+  wip: Snapshot<T> | EmptyWIP;
```

Comment:
given we use this once
```suggestion
  wip: Snapshot<T> | typeof EMPTY_WIP;
```